main.py

```diff
@@ -1,10 +1,54 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_info = json.load(file)
+
+    for player, data in players_info.items():
+        race_info = data["race"]
+        if race_info:
+            if not Race.objects.filter(name=race_info["name"]).exists():
+                race = Race.objects.create(
+                    name=race_info["name"],
+                    description=race_info["description"]
+                )
+            else:
+                race = Race.objects.get(name=race_info["name"])
+
+        skill_info = race_info["skills"]
+        if skill_info:
```

Comment:
Since absent skills are represented as an empty list, this check is not necessary.